### PR TITLE
Zope 2.13: fixup! Add change log entry for #395

### PR DESCRIPTION
**Pull request by perrinjerome**
That changelog entry was referencing another PR

### CODE REVIEW

File: doc/CHANGES.rst [hunk 1]
```diff
@@ -19,8 +19,8 @@ Security related fixes
 Features
 ++++++++
 
-- Add wildcard rewrite to sub host name in virtualHostMonster.
-  (`#317 <https://github.com/zopefoundation/Zope/issues/317>`_)
+- Add support of IPv6 named hosts in VirtualHostMonster
+  (`#395 <https://github.com/zopefoundation/Zope/issues/395>`_)
```
Comment (perrinjerome) [inline doc/CHANGES.rst]: Oh wait, the link is also wrong, even if github will redirect the #395 PR, there was no issue opened ( except the old one on launchpad https://bugs.launchpad.net/zope2/+bug/699865  but I guess we don't need to reference it here )

Comment (perrinjerome) [reply doc/CHANGES.rst]: I just amended this commit.

As I said on https://github.com/zopefoundation/Zope/pull/400 , it's unclear if this should be listed in Bugfixes or new features (but let's discuss this in https://github.com/zopefoundation/Zope/pull/400 )

**Comment by icemac** [comment]
@perrinjerome  Thank you for bringing this up I got the issues mixed up.

**Comment by perrinjerome** [comment]
@icemac I updated this PR for 2.13 branch with the same change log text that we added on master in https://github.com/zopefoundation/Zope/pull/400, please take another look.

One difference between change log entry on 2.13 and on master is the link to the relevant PR, because we had two different PRs.

Thanks again for your guidance and comments in all this.

**Pull request merged**